lib/dbservice/batches/batch.ex

```diff
@@ -10,6 +10,7 @@ defmodule Dbservice.Batches.Batch do
   schema "batch" do
     field :name, :string
     field :contact_hours_per_week, :integer
+    field :batch_id, :string
```

Comment:
Remember how we talked about having a parent batch as well? Maybe you can add that change to this PR only, since you are anyway altering the batch table.

Reply:
The migration created for this has already ran in staging. Let's make a separate PR for this